helm/alfresco-content-services/charts/alfresco-elasticsearch-connector/values.yaml

```diff
@@ -0,0 +1,57 @@
+image:
+  repositoryPrefix: quay.io/alfresco/alfresco-elasticsearch
```

Comment:
please use explicit repository and tag variables per each image in order to keep FluxCD happy without extra configuration